docs/operations/automatic-data-locality.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Automatic data locality"
+linkTitle: "Automatic data locality"
+---
+
+## Ondat volumes move with their pods
+
+Having an Ondat volume instance on the same node as the pod using it can bring some benifits to your application. Unfortunatly pods can move around over time leaving the Volume behind.
+
+Automatic-Data-Locality fights that, making the master Ondat volume instance follow the pod as it moves between cluster nodes.
```

Comment:
I'd probably use `mitigate` instead of `fights. 

Also, I believe we should use "primary" instead of "master" for public documentation.

Reply:
Replaced